zeebe/backup-stores/azure/src/main/java/io/camunda/zeebe/backup/azure/AzureBackupConfig.java

```diff
@@ -23,6 +24,7 @@ public static class Builder {
 
     // maps to the basePath env variable
     private String containerName;
+    private String auth;
```

Comment:
:wrench: Either define an enum for it or declare "auth" as a static field. Currently it's hard-coded in at least two places that could go out of sync eventually.